cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateROPaths(spec *rspec.Spec) error {
+	fmt.Println("validating Readonly Paths")
```

Comment:
using 'validating readonlyPaths' as Mrunal suggested [match what is expected to be the property name]( https://github.com/opencontainers/ocitools/pull/70).